*** tests/A-01-08-M-ok.out	2021-10-25 09:43:44.150005695 +0100
--- tests/A-01-08-M-ok.outhyp	2021-10-28 12:57:25.861879372 +0100
***************
*** 62,68 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Menu Principal
  1 - Abrir
  2 - Guardar
  3 - Mostrar Data
--- 62,68 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu Principal
  1 - Abrir
  2 - Guardar
  3 - Mostrar Data
***************
*** 73,76 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: 
\ No newline at end of file
--- 73,76 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Fim de entrada de dados (R-EOF): pt.tecnico.uilib.text.RuntimeEOFException: Fim do fluxo de dados de entrada
