*** tests/A-01-09-M-ok.out	2021-10-25 09:43:44.150005695 +0100
--- tests/A-01-09-M-ok.outhyp	2021-10-28 15:50:03.179217525 +0100
***************
*** 20,26 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Data actual: 20
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 20,26 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Data actual: 0
  Menu Principal
  1 - Abrir
  2 - Guardar
