*** tests/A-03-03-M-ok.out	2021-10-15 15:30:49.000000000 +0100
--- tests/A-03-03-M-ok.outhyp	2021-10-26 21:46:35.869172766 +0100
***************
*** 15,24 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: HIDROGENIO|200|5000
! ROLHA|2000|500
! SAL|1000|500
! VIDRO|1000|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
--- 15,24 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: ROLHA|0.0|stock-atual-total
! VIDRO|0.0|stock-atual-total
! HIDROGENIO|0.0|stock-atual-total
! SAL|0.0|stock-atual-total
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
