*** tests/A-03-02-M-ok.out	2021-10-15 13:02:20.000000000 +0100
--- tests/A-03-02-M-ok.outhyp	2021-10-26 12:13:44.741183994 +0100
***************
*** 15,22 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: HIDROGENIO|200|5000
! Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
  3 - Visualizar lotes fornecidos por parceiro
--- 15,21 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
  3 - Visualizar lotes fornecidos por parceiro
