*** tests/A-04-02-M-ok.out	2021-10-15 19:42:43.000000000 +0100
--- tests/A-04-02-M-ok.outhyp	2021-10-26 21:26:22.503055481 +0100
***************
*** 15,21 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: HIDROGENIO|S1|200|5000
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
--- 15,21 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: S1|HIDROGENIO|200.0|5000
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
