*** tests/A-04-04-M-ok.out	2021-10-25 09:43:44.150005695 +0100
--- tests/A-04-04-M-ok.outhyp	2021-10-27 11:27:08.809463775 +0100
***************
*** 15,31 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: ROLHA|M1|2000|500
! ROLHA|S1|100|100
! ROLHA|S2|1000|100
! ROLHA|S3|500|100
! TRALHA|M1|20|5000
! TRALHA|M1|200|500
! TRALHA|M1|2000|500
! TRALHA|S1|100|100
! TRALHA|S2|100|100
! TRALHA|S3|50|100
! Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
  3 - Visualizar lotes fornecidos por parceiro
--- 15,21 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
  3 - Visualizar lotes fornecidos por parceiro
