*** tests/A-06-04-M-ok.out	2021-10-25 23:03:50.208294000 +0100
--- tests/A-06-04-M-ok.outhyp	2021-11-15 15:35:23.201300166 +0000
***************
*** 15,26 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: Identificador do produto: HIDROGENIO|M1|2|5000
! HIDROGENIO|M1|20|5000
  HIDROGENIO|M1|30|5000
- HIDROGENIO|S1|20|5000
  HIDROGENIO|S1|50|5000
! HIDROGENIO|S1|200|5000
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
--- 15,26 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: Identificador do produto: HIDROGENIO|S1|200|5000
! HIDROGENIO|M1|2|5000
  HIDROGENIO|M1|30|5000
  HIDROGENIO|S1|50|5000
! HIDROGENIO|S1|20|5000
! HIDROGENIO|M1|20|5000
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
