*** tests/A-08-01-M-ok.out	2021-10-25 09:43:44.150005695 +0100
--- tests/A-08-01-M-ok.outhyp	2021-10-27 11:27:09.265470212 +0100
***************
*** 17,28 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|0|0|0
! P1|Lages do Chão|Lisboa, Portugal|NORMAL|0|0|0|0
! P3|Brian Rocks|Koln, Germany|NORMAL|0|0|0|0
! R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|0|0|0|0
! S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! W2|Pedraria Fonseca|Oeiras, Portugal|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 17,28 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|Normal|0.0|0|0|0
! P1|Lages do Chão|Lisboa, Portugal|Normal|0.0|0|0|0
! P3|Brian Rocks|Koln, Germany|Normal|0.0|0|0|0
! R2|Jorge Figueiredo|Lisboa, Portugal|Normal|0.0|0|0|0
! S1|Toshiba|Tokyo, Japan|Normal|0.0|0|0|0
! W2|Pedraria Fonseca|Oeiras, Portugal|Normal|0.0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
