*** tests/A-09-01-M-ok.out	2021-10-14 19:21:55.000000000 +0100
--- tests/A-09-01-M-ok.outhyp	2021-10-26 12:13:45.497186792 +0100
***************
*** 25,31 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: aaa|dwd|ende@sfsdf-com|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 25,31 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: aaa|dwd|ende@sfsdf-com|Normal|0.0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
