*** tests/A-10-16-M-ok.out	2021-11-05 13:01:34.809548000 +0000
--- tests/A-10-16-M-ok.outhyp	2021-11-15 15:35:26.913305376 +0000
***************
*** 141,146 ****
--- 141,150 ----
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  NEW|SAL|20
+ NEW|SAL|20
+ NEW|SAL|20
+ NEW|SAL|20
+ NEW|SAL|20
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
***************
*** 150,156 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! NEW|SAL|8
  NEW|SAL|20
  Gestão de Parceiros
  1 - Mostrar parceiro
--- 154,160 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! NEW|SAL|20
  NEW|SAL|20
  Gestão de Parceiros
  1 - Mostrar parceiro
