*** tests/A-10-02-M-ok.out	2021-11-08 09:39:22.340214323 +0000
--- tests/A-10-02-M-ok.outhyp	2021-11-11 15:17:30.114820469 +0000
***************
*** 17,24 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Identificador do produto: Activa/desactivar notificações de um produto: Operação inválida: O produto 'DEDE' não existe.
! Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
  3 - Registar parceiro
--- 17,23 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
  3 - Registar parceiro
***************
*** 26,33 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Identificador do produto: Activa/desactivar notificações de um produto: Operação inválida: O parceiro 'MM1' não existe.
! Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
  3 - Registar parceiro
--- 25,33 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Número inválido!
! Escolha uma opção: Número inválido!
! Escolha uma opção: Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
  3 - Registar parceiro
***************
*** 35,40 ****
--- 35,42 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
+ Escolha uma opção: Número inválido!
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu Principal
  1 - Abrir
  2 - Guardar
