*** tests/A-10-13-M-ok.out	2021-11-05 13:01:33.837853000 +0000
--- tests/A-10-13-M-ok.outhyp	2021-11-15 15:35:26.405304669 +0000
***************
*** 61,67 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! NEW|SAL|5
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 61,67 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! NEW|SAL|20
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
***************
*** 71,77 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! NEW|SAL|5
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 71,77 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! NEW|SAL|20
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
