*** tests/A-10-08-M-ok.out	2021-11-05 13:01:32.123886000 +0000
--- tests/A-10-08-M-ok.outhyp	2021-11-15 15:35:25.721303714 +0000
***************
*** 147,153 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! BARGAIN|SAL|50
  BARGAIN|SAL|2
  Gestão de Parceiros
  1 - Mostrar parceiro
--- 147,157 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! BARGAIN|SAL|2
! BARGAIN|SAL|2
! BARGAIN|SAL|2
! BARGAIN|SAL|2
! BARGAIN|SAL|2
  BARGAIN|SAL|2
  Gestão de Parceiros
  1 - Mostrar parceiro
***************
*** 158,165 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! BARGAIN|SAL|50
! BARGAIN|SAL|10
  BARGAIN|SAL|2
  Gestão de Parceiros
  1 - Mostrar parceiro
--- 162,169 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! BARGAIN|SAL|2
! BARGAIN|SAL|2
  BARGAIN|SAL|2
  Gestão de Parceiros
  1 - Mostrar parceiro
