*** tests/A-11-02-M-ok.out	2021-11-07 16:04:59.000000000 +0000
--- tests/A-11-02-M-ok.outhyp	2021-11-10 22:10:01.866735794 +0000
***************
*** 41,47 ****
  4 - Visualizar lotes de um dado produto
  0 - Sair
  Escolha uma opção: HIDROGENIO|200|5000
! ROLHA|30|1100
  SAL|1000|500
  VIDRO|1000|500
  Gestão de Produtos
--- 41,47 ----
  4 - Visualizar lotes de um dado produto
  0 - Sair
  Escolha uma opção: HIDROGENIO|200|5000
! ROLHA|2500|1100
  SAL|1000|500
  VIDRO|1000|500
  Gestão de Produtos
***************
*** 52,59 ****
  0 - Sair
  Escolha uma opção: HIDROGENIO|S1|200|5000
  ROLHA|M1|20|500
- ROLHA|M1|25|100
  ROLHA|M1|30|500
  SAL|S1|1000|500
  VIDRO|M1|1000|500
  Gestão de Produtos
--- 52,59 ----
  0 - Sair
  Escolha uma opção: HIDROGENIO|S1|200|5000
  ROLHA|M1|20|500
  ROLHA|M1|30|500
+ ROLHA|M1|2500|100
  SAL|S1|1000|500
  VIDRO|M1|1000|500
  Gestão de Produtos
