*** tests/A-12-04-M-ok.out	2021-11-08 09:39:22.340214323 +0000
--- tests/A-12-04-M-ok.outhyp	2021-11-11 13:04:14.263082140 +0000
***************
*** 62,69 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: -60000
! Saldo contabilístico: -60000
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 62,69 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: -2300
! Saldo contabilístico: -2300
  Menu Principal
  1 - Abrir
  2 - Guardar
