*** tests/A-12-03-M-ok.out	2021-11-07 16:04:59.000000000 +0000
--- tests/A-12-03-M-ok.outhyp	2021-11-11 13:41:31.309083895 +0000
***************
*** 120,127 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: -60000
! Saldo contabilístico: -60000
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 120,127 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: -2300
! Saldo contabilístico: -2300
  Menu Principal
  1 - Abrir
  2 - Guardar
