*** tests/A-12-04-M-ok.out	2021-11-07 16:04:59.000000000 +0000
--- tests/A-12-04-M-ok.outhyp	2021-11-10 20:03:23.348982265 +0000
***************
*** 62,69 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: -60000
! Saldo contabilístico: -60000
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 62,68 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: -60000
  Menu Principal
  1 - Abrir
  2 - Guardar
