*** tests/A-12-02-M-ok.out	2021-11-07 16:04:59.000000000 +0000
--- tests/A-12-02-M-ok.outhyp	2021-11-11 13:41:31.229083931 +0000
***************
*** 34,41 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: -20000
! Saldo contabilístico: -20000
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 34,41 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: -1000
! Saldo contabilístico: -1000
  Menu Principal
  1 - Abrir
  2 - Guardar
