*** tests/A-13-03-M-ok.out	2021-11-07 16:04:59.000000000 +0000
--- tests/A-13-03-M-ok.outhyp	2021-11-09 15:07:46.220722900 +0000
***************
*** 13,24 ****
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Preço: HIDROGENIO|S1|200|5000
! ROLHA|M1|30|500
! ROLHA|R1|20|500
! SAL|S1|1000|500
! VIDRO|M1|1000|500
! Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
--- 13,19 ----
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Preço: Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
