*** tests/A-13-02-M-ok.out	2021-11-07 16:04:59.000000000 +0000
--- tests/A-13-02-M-ok.outhyp	2021-11-11 13:41:31.585083773 +0000
***************
*** 51,60 ****
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Preço: HIDROGENIO|S1|2|5000
! ROLHA|M1|2|500
! VIDRO|M1|1|500
! Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
--- 51,57 ----
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Preço: Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
