*** tests/A-14-02-M-ok.out	2021-11-07 16:04:59.000000000 +0000
--- tests/A-14-02-M-ok.outhyp	2021-11-11 21:00:50.000849635 +0000
***************
*** 48,54 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: COMPRA|0|M1|SAL|10|100|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 48,54 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: COMPRA|1|M1|SAL|10|100|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 71,77 ****
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
--- 71,78 ----
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: COMPRA|1|M1|VIDRO|20|200|0
! Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
