*** tests/A-14-05-M-ok.out	2021-11-01 23:11:20.620093000 +0000
--- tests/A-14-05-M-ok.outhyp	2021-11-15 15:35:31.217311285 +0000
***************
*** 109,116 ****
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: VENDA|1|M1|ROLHA|3|6|6|3|2
! VENDA|4|M1|SAL|5|30|30|3|2
  Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
--- 109,116 ----
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: VENDA|1|M1|ROLHA|3|9|9|3|2
! VENDA|4|M1|SAL|5|70|70|3|2
  Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
