*** tests/A-14-01-M-ok.out	2021-11-07 16:04:59.000000000 +0000
--- tests/A-14-01-M-ok.outhyp	2021-11-10 01:03:59.777880080 +0000
***************
*** 13,36 ****
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Facturas pagas por parceiro: Operação inválida: O parceiro 'nãoexiste' não existe.
! Consultas
! 1 - Produtos com Preço Abaixo de Limite
! 2 - Facturas pagas por parceiro
! 0 - Sair
! Escolha uma opção: Identificador do parceiro: Consultas
! 1 - Produtos com Preço Abaixo de Limite
! 2 - Facturas pagas por parceiro
! 0 - Sair
! Escolha uma opção: Menu Principal
! 1 - Abrir
! 2 - Guardar
! 3 - Mostrar Data
! 4 - Avançar Data
! 5 - Gestão de Produtos
! 6 - Gestão de Parceiros
! 7 - Gestão de Transacções
! 8 - Menu de Consultas
! 9 - Ver Saldo Global
! 0 - Sair
! Escolha uma opção: 
\ No newline at end of file
--- 13,16 ----
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: 
\ No newline at end of file
