*** tests/A-15-03-M-ok.out	2021-10-31 18:59:44.776533000 +0000
--- tests/A-15-03-M-ok.outhyp	2021-11-15 15:35:31.737311989 +0000
***************
*** 48,54 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|S1|200|499
  HIDROGENIO|S1|200|5000
  ROLHA|M1|2|1
  ROLHA|R1|2|500
--- 48,54 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|S1|200|500
  HIDROGENIO|S1|200|5000
  ROLHA|M1|2|1
  ROLHA|R1|2|500
***************
*** 80,86 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: M1|Rohit Figueiredo|New Delhi, India|NORMAL|980|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 80,86 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|0|98|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
