*** tests/A-15-10-M-ok.out	2021-10-31 23:57:26.927950000 +0000
--- tests/A-15-10-M-ok.outhyp	2021-11-15 15:35:32.785313403 +0000
***************
*** 48,54 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|S1|100|5
  HIDROGENIO|S1|200|5000
  ROLHA|R1|2|10
  ROLHA|R1|2|500
--- 48,54 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|S1|100|15
  HIDROGENIO|S1|200|5000
  ROLHA|R1|2|10
  ROLHA|R1|2|500
***************
*** 62,68 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|S1|100|5
  HIDROGENIO|S1|200|5000
  ROLHA|R1|2|10
  ROLHA|R1|2|500
--- 62,68 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|S1|100|15
  HIDROGENIO|S1|200|5000
  ROLHA|R1|2|10
  ROLHA|R1|2|500
***************
*** 76,82 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|100|5|ROLHA:1#SAL:5#VIDRO:10
  HIDROGENIO|200|5000
  ROLHA|2|510
  SAL|16|550
--- 76,82 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|100|15|ROLHA:1#SAL:5#VIDRO:10
  HIDROGENIO|200|5000
  ROLHA|2|510
  SAL|16|550
