*** tests/A-16-02-M-ok.out	2021-11-08 09:39:22.340214323 +0000
--- tests/A-16-02-M-ok.outhyp	2021-11-11 15:17:32.926870625 +0000
***************
*** 67,73 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|S1|SAL|5|25|25|10|9
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 67,73 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|S1|SAL|5|25|25|10|null
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 94,100 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|250|0|25|25
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 94,100 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|25|23
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
