*** tests/A-19-05-M-ok.out	2021-11-11 14:06:37.815183963 +0000
--- tests/A-19-05-M-ok.outhyp	2021-11-11 23:00:30.821555685 +0000
***************
*** 16,22 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Data limite de pagamento: Identificador do produto: Quantidade: Registar venda: Operação inválida: Produto 'ROLHA': pedido=80, existências=75
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 16,22 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Data limite de pagamento: Identificador do produto: Quantidade: Registar venda: Operação inválida: Produto 'GARRAFA': pedido=10, existências=2
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
