*** tests/A-20-15-M-ok.out	2021-11-11 16:05:31.855031000 +0000
--- tests/A-20-15-M-ok.outhyp	2021-11-15 15:35:37.945320244 +0000
***************
*** 119,125 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|3|M1|SAL|3|300|300|1|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 119,125 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|3|M1|SAL|3|300|270|1|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
