*** testsFinal/A-04-08-M-ok.out	2021-10-24 17:46:04.420193000 +0100
--- testsFinal/A-04-08-M-ok.outhyp	2021-11-15 15:39:14.329471562 +0000
***************
*** 44,49 ****
--- 44,50 ----
  4 - Visualizar lotes de um dado produto
  0 - Sair
  Escolha uma opção: Identificador do parceiro: HIDROGENIO|S1|2|5000
+ SAL|S1|1|0
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
