*** testsFinal/A-05-08-M-ok.out	2021-10-25 12:13:20.493544000 +0100
--- testsFinal/A-05-08-M-ok.outhyp	2021-11-15 15:39:15.189471767 +0000
***************
*** 16,31 ****
  4 - Visualizar lotes de um dado produto
  0 - Sair
  Escolha uma opção: Identificador do parceiro: ROLHA|M1|2|500
  ROLHA|M1|20|100
  ROLHA|M1|100|10
- ROLHA|M1|200|1000
- ROLHA|M1|400|100
- TRALHA|M1|5|500
  TRALHA|M1|10|100
  TRALHA|M1|40|100
  TRALHA|M1|50|10
  TRALHA|M1|70|410
  TRALHA|M1|100|50
  TRALHA|M1|2000|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
--- 16,31 ----
  4 - Visualizar lotes de um dado produto
  0 - Sair
  Escolha uma opção: Identificador do parceiro: ROLHA|M1|2|500
+ TRALHA|M1|5|500
  ROLHA|M1|20|100
  ROLHA|M1|100|10
  TRALHA|M1|10|100
  TRALHA|M1|40|100
  TRALHA|M1|50|10
+ ROLHA|M1|200|1000
  TRALHA|M1|70|410
  TRALHA|M1|100|50
+ ROLHA|M1|400|100
  TRALHA|M1|2000|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
