*** testsFinal/A-10-07-M-ok.out	2021-11-05 13:01:31.650599000 +0000
--- testsFinal/A-10-07-M-ok.outhyp	2021-11-15 15:39:16.585472096 +0000
***************
*** 95,101 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! BARGAIN|SAL|50
  BARGAIN|SAL|10
  Gestão de Parceiros
  1 - Mostrar parceiro
--- 95,101 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! BARGAIN|SAL|10
  BARGAIN|SAL|10
  Gestão de Parceiros
  1 - Mostrar parceiro
***************
*** 106,112 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! BARGAIN|SAL|50
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 106,114 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! BARGAIN|SAL|10
! BARGAIN|SAL|10
! BARGAIN|SAL|10
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
