*** testsFinal/A-10-20-M-ok.out	2021-11-05 13:01:36.152472000 +0000
--- testsFinal/A-10-20-M-ok.outhyp	2021-11-15 15:39:18.449472526 +0000
***************
*** 180,187 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! NEW|SAL|10
! BARGAIN|ROLHA|100
  NEW|SAL|20
  BARGAIN|ROLHA|50
  Gestão de Parceiros
--- 180,187 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! NEW|SAL|20
! BARGAIN|ROLHA|50
  NEW|SAL|20
  BARGAIN|ROLHA|50
  Gestão de Parceiros
***************
*** 193,200 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! NEW|SAL|10
! BARGAIN|ROLHA|100
  NEW|SAL|20
  BARGAIN|ROLHA|50
  Gestão de Parceiros
--- 193,200 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! NEW|SAL|20
! BARGAIN|ROLHA|50
  NEW|SAL|20
  BARGAIN|ROLHA|50
  Gestão de Parceiros
***************
*** 206,213 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M3|M3|M3 add|NORMAL|0|0|0|0
- NEW|SAL|20
- BARGAIN|ROLHA|50
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 206,211 ----
***************
*** 217,224 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M4|M4|M4 end|NORMAL|0|0|0|0
- NEW|SAL|20
- BARGAIN|ROLHA|50
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 215,220 ----
