*** testsFinal/A-10-15-M-ok.out	2021-11-05 13:01:34.449182000 +0000
--- testsFinal/A-10-15-M-ok.outhyp	2021-11-15 15:39:17.745472365 +0000
***************
*** 118,123 ****
--- 118,125 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
+ NEW|SAL|8
+ NEW|SAL|8
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
