*** testsFinal/A-11-11-M-ok.out	2021-10-26 19:36:30.487098000 +0100
--- testsFinal/A-11-11-M-ok.outhyp	2021-11-15 15:39:19.765472823 +0000
***************
*** 131,138 ****
  4 - Visualizar lotes de um dado produto
  0 - Sair
  Escolha uma opção: Identificador do produto: VIDRO|M1|30|500
- VIDRO|M1|40|50
  VIDRO|M2|100|10
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
--- 131,138 ----
  4 - Visualizar lotes de um dado produto
  0 - Sair
  Escolha uma opção: Identificador do produto: VIDRO|M1|30|500
  VIDRO|M2|100|10
+ VIDRO|M1|40|50
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
