*** testsFinal/A-12-08-M-ok.out	2021-11-12 10:27:26.154535000 +0000
--- testsFinal/A-12-08-M-ok.outhyp	2021-11-15 20:27:48.535308633 +0000
***************
*** 77,84 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 120
! Saldo contabilístico: 600
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 77,84 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 20
! Saldo contabilístico: 500
  Menu Principal
  1 - Abrir
  2 - Guardar
