*** testsFinal/A-15-10-M-ok.out	2021-11-17 15:27:39.642684500 +0000
--- testsFinal/A-15-10-M-ok.outhyp	2021-11-17 15:58:15.564057382 +0000
***************
*** 106,112 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: R1|António Figueiredo|Lisboa|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 106,112 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: R1|António Figueiredo|Lisboa|NORMAL|-8200|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
