*** testsFinal/A-15-09-M-ok.out	2021-11-17 15:27:39.642684500 +0000
--- testsFinal/A-15-09-M-ok.outhyp	2021-11-17 15:58:15.408053079 +0000
***************
*** 23,29 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: DESAGREGAÇÃO|0|R1|GARRAFA|9|362|362|0|ROLHA:9:18#SAL:45:720#VIDRO:90:900
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 23,29 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: DESAGREGAÇÃO|0|R1|GARRAFA|9|1962|1962|0|ROLHA:9:180#SAL:45:1170#VIDRO:90:2700
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
