*** testsFinal/A-15-04-M-ok.out	2021-10-31 18:58:50.933418000 +0000
--- testsFinal/A-15-04-M-ok.outhyp	2021-11-17 04:24:19.537896862 +0000
***************
*** 48,54 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|S1|200|495
  HIDROGENIO|S1|200|5000
  ROLHA|M1|2|5
  ROLHA|R1|2|500
--- 48,54 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|S1|200|500
  HIDROGENIO|S1|200|5000
  ROLHA|M1|2|5
  ROLHA|R1|2|500
***************
*** 81,87 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: M1|Rohit Figueiredo|New Delhi, India|NORMAL|900|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 81,87 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|0|90|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
