*** testsFinal/A-15-05-M-ok.out	2021-10-31 21:59:41.505087000 +0000
--- testsFinal/A-15-05-M-ok.outhyp	2021-11-17 04:24:19.677896878 +0000
***************
*** 48,54 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|200|0|ROLHA:1#SAL:5#VIDRO:10
  HIDROGENIO|200|5000
  ROLHA|2|505
  SAL|16|525
--- 48,54 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|200|5|ROLHA:1#SAL:5#VIDRO:10
  HIDROGENIO|200|5000
  ROLHA|2|505
  SAL|16|525
***************
*** 59,65 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: HIDROGENIO|S1|200|5000
  ROLHA|R1|2|5
  ROLHA|R1|2|500
  SAL|R1|16|25
--- 59,66 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|S1|200|5
! HIDROGENIO|S1|200|5000
  ROLHA|R1|2|5
  ROLHA|R1|2|500
  SAL|R1|16|25
***************
*** 91,97 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: R1|António Figueiredo|Lisboa|NORMAL|900|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 92,98 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: R1|António Figueiredo|Lisboa|NORMAL|0|0|90|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
