*** testsFinal/A-19-08-M-ok.out	2021-11-06 08:31:15.275004000 +0000
--- testsFinal/A-19-08-M-ok.outhyp	2021-11-19 22:27:50.035893761 +0000
***************
*** 48,55 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: CORTIÇA|M1|1|1400
! PLASTICO|M1|1|980
  SAL|S1|100|500
  VIDRO|M1|1|190
  Gestão de Produtos
--- 48,56 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: CORTIÇA|M1|1|1300
! PLASTICO|M1|1|960
! ROLHA|M1|7|20
  SAL|S1|100|500
  VIDRO|M1|1|190
  Gestão de Produtos
