*** testsFinal/A-19-10-M-ok.out	2021-11-06 09:26:02.136851000 +0000
--- testsFinal/A-19-10-M-ok.outhyp	2021-11-15 20:27:53.035312991 +0000
***************
*** 23,29 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|M2|ROLHA|120|330|330|2
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 23,29 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|M2|ROLHA|120|730|730|2
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
