*** testsFinal/A-19-07-M-ok.out	2021-11-11 09:51:24.671709000 +0000
--- testsFinal/A-19-07-M-ok.outhyp	2021-11-18 06:28:03.199874001 +0000
***************
*** 64,81 ****
  4 - Visualizar lotes de um dado produto
  0 - Sair
  Escolha uma opção: GARRAFA|39|0|ROLHA:10#VIDRO:5
! ROLHA|2|35
  SAL|40|500
! VIDRO|3|5
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: ROLHA|M1|2|35
  SAL|S1|40|500
! VIDRO|M1|3|5
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
--- 64,81 ----
  4 - Visualizar lotes de um dado produto
  0 - Sair
  Escolha uma opção: GARRAFA|39|0|ROLHA:10#VIDRO:5
! ROLHA|2|5
  SAL|40|500
! VIDRO|3|0
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: ROLHA|M1|2|5
  SAL|S1|40|500
! VIDRO|M1|3|0
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
