*** testsFinal/A-20-14-M-ok.out	2021-11-17 15:27:39.650684821 +0000
--- testsFinal/A-20-14-M-ok.outhyp	2021-11-17 15:58:20.724199917 +0000
***************
*** 75,81 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|SELECTION|4170|0|440|417
  M2|Toshiba|Tokyo, Japan|SELECTION|2100|0|210|210
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
--- 75,81 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|SELECTION|6270|0|440|627
  M2|Toshiba|Tokyo, Japan|SELECTION|2100|0|210|210
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
***************
*** 145,151 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|SELECTION|24870|0|2740|2487
  M2|Toshiba|Tokyo, Japan|SELECTION|2100|0|210|210
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
--- 145,151 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|ELITE|26970|0|2740|2697
  M2|Toshiba|Tokyo, Japan|SELECTION|2100|0|210|210
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
***************
*** 215,221 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|ELITE|25230|0|2780|2523
  M2|Toshiba|Tokyo, Japan|SELECTION|2100|0|210|210
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
--- 215,221 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|ELITE|27330|0|2780|2733
  M2|Toshiba|Tokyo, Japan|SELECTION|2100|0|210|210
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
