*** testsFinal/A-20-09-M-ok.out	2021-11-11 10:45:01.002055000 +0000
--- testsFinal/A-20-09-M-ok.outhyp	2021-11-15 15:39:27.941474551 +0000
***************
*** 59,65 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|M1|SAL|28|2800|4480|2
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 59,65 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|M1|SAL|28|2800|2800|2
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 67,73 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|1|M2|SAL|28|2800|3220|5
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 67,73 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|1|M2|SAL|28|2800|2800|5
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 89,95 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|1|M2|SAL|28|2800|3220|5|8
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 89,95 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|1|M2|SAL|28|2800|2800|5|8
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 97,103 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|M1|SAL|28|2800|4480|2|8
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 97,103 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|M1|SAL|28|2800|2800|2|8
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 124,131 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|0|2800|4480
! M2|Toshiba|Tokyo, Japan|NORMAL|0|0|2800|3220
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
--- 124,131 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|0|2800|2800
! M2|Toshiba|Tokyo, Japan|NORMAL|0|0|2800|2800
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
